src/Apps/Order/Routes/Shipping/useAddressAutocomplete.ts

```diff
@@ -0,0 +1,135 @@
+import { AutocompleteInputOptionType } from "@artsy/palette"
+import { Address } from "Components/AddressForm"
+import { useDebounce } from "Utils/Hooks/useDebounce"
+import { getENV } from "Utils/getENV"
+import { useCallback, useState } from "react"
+
+// NOTE: Due to the format of this key (a long string of numbers that cannot be parsed as json)
+// This key must be set in the env as a json string like SMARTY_EMBEDDED_KEY_JSON_JSON='{ "key": "xxxxxxxxxxxxxxxxxx" }'
```

Comment:
Add to .env.example maybe? Not necessary for development though.
```suggestion
// This key must be set in the env as a json string like SMARTY_EMBEDDED_KEY_JSON_JSON={ "key": "xxxxxxxxxxxxxxxxxx" }
```